docs/generated/CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
 ## [1.1.1](https://github.com/foobaragency/cf-migrations/compare/v1.1.0...v1.1.1) (2021-08-02)
 
-
 ### Bug Fixes
 
-* don't save migrations that were aborted ([7352809](https://github.com/foobaragency/cf-migrations/commit/73528099e7ae087fd6d0900c02202cb17a45bf97))
-* duplicate entry in migrationResults ([e18d159](https://github.com/foobaragency/cf-migrations/commit/e18d1598b63f495e5a8d5f5e628515c6dcff77b4))
+- don't save migrations that were aborted ([7352809](https://github.com/foobaragency/cf-migrations/commit/73528099e7ae087fd6d0900c02202cb17a45bf97))
```

Comment:
These changes are prettier fixes and this document is generated during the Release workflow. We should adjust the settings and avoid fixing such issues after every release.